src/abstractdataframe/reshape.jl

```diff
@@ -258,7 +258,7 @@ function _unstack(df::AbstractDataFrame, rowkey::Int,
 end
 
 unstack(df::AbstractDataFrame, rowkey::ColumnIndex,
-        colkey::ColumnIndex, value::ColumnIndex) =
+        colkey::Int, value::Int) =
```

Comment:
Why do `index(df)[rowkey]` if we know the index is an integer already?

Reply:
`rowkey` can be a `Symbol`, but we can drop it for `colkey` and `value`. Fixed.